### PR TITLE
8334441: Mark tests in jdk_security_infra group as manual

### PR DESCRIPTION
**Pull request by rhalade**
Updated all the tests that depend on external infrastructure services as manual. These tests may fail with external reasons, for instance - change in CA test portal, certificate status updates, or network issues.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8334441](https://bugs.openjdk.org/browse/JDK-8334441): Mark tests in jdk_security_infra group as manual (**Bug** - P3)

### Reviewers
 * [Christoph Langer](https://openjdk.org/census#clanger) (@RealCLanger - **Reviewer**) ⚠️ Review applies to [5f1889af](https://git.openjdk.org/jdk/pull/19814/files/5f1889afe9a3ec3f2230ad631e5d49b84fd19d76)
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19814/head:pull/19814` \
`$ git checkout pull/19814`

Update a local copy of the PR: \
`$ git checkout pull/19814` \
`$ git pull https://git.openjdk.org/jdk.git pull/19814/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19814`

View PR using the GUI difftool: \
`$ git pr show -t 19814`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19814.diff">https://git.openjdk.org/jdk/pull/19814.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19814#issuecomment-2181308065)

### CODE REVIEW

File: test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java [hunk 1]
```diff
@@ -27,12 +27,12 @@
  * @summary Interoperability tests with Actalis CA
  * @library /test/lib
  * @build jtreg.SkippedException ValidatePathWithURL CAInterop
- * @run main/othervm -Djava.security.debug=certpath,ocsp
+ * @run main/othervm/manual/manual -Djava.security.debug=certpath,ocsp
```
Comment (RealCLanger) [inline test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java:30]: What is the reason why manual is written twice? (Here and in all other places in this file...)

**Review by RealCLanger — APPROVED** [review]
Looks good, although I don't see why sometimes the directive is @run main/othervm/manual/manual (double occurence of manual).

File: test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java (hunk 1 above)
Comment (rhalade) [reply test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java]: good catch! I have fixed it.

**Comment by rhalade** [comment]
> Is there still going to be some monitoring of these tests? Making it harder to run these tests potentially means that genuine failures can go unnoticed and JDK certificates are not updated when needed.

Tracking won’t be any different than other manual tests in the test suite. Within Oracle, we recommend that developers working on security libraries fixes run these tests before integration. All manual tests are definitely run during the release certification.

**Pull request closed without merging**